### PR TITLE
Nuxt slice templates

### PR DESCRIPTION
**Pull request by hypervillain**
closes https://linear.app/prismic/issue/DT-1631/aauser-i-have-access-to-a-nuxt-slice-template

### CODE REVIEW

File: packages/adapter-next/src/plugin.ts [hunk 1]
```diff
@@ -177,7 +177,7 @@ export const plugin = defineSliceMachinePlugin<PluginOptions>({
 				templates: [Hero, CustomerLogos, AlternateGrid, CallToAction],
 				componentFileNames: {
 					js: "javascript.jsx",
-					ts: "javascript.tsx",
+					ts: "typescript.tsx",
```
Comment (hypervillain) [inline packages/adapter-next/src/plugin.ts:180]: this fixes a bug in previous PR

File: packages/adapter-nuxt/public/Hero/javascript.vue [hunk 2]
```diff
@@ -0,0 +1,225 @@
+<script setup>
+import { isFilled } from "@prismicio/client";
+
+defineProps(getSliceComponentProps(["slice", "index", "slices", "context"]));
+</script>
+
+<template>
+	<section
```
Comment (xrutayisire) [inline packages/adapter-nuxt/public/Hero/javascript.vue:8]: You don't want to set the same 2 spaces indent as we did for Next?

File: packages/adapter-nuxt/public/Hero/javascript.vue [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+	.es-fullpage-hero__content {
+		flex-direction: row;
+	}
+
+	.es-fullpage-hero__content > div {
+		width: 50%;
+	}
+}
+
+.es-fullpage-hero__content-right__intro {
```
Comment (xrutayisire) [inline packages/adapter-nuxt/public/Hero/javascript.vue]: Not used right? Maybe we can check other Hero from Next / Svelte

File: packages/adapter-nuxt/public/Hero/javascript.vue [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+		</div>
+		<div class="es-fullpage-hero__content-right">
+			<div class="es-fullpage-hero__content__intro">
+				<p
+					v-if="isFilled.keyText(slice.primary.eyebrowHeadline)"
+					class="es-fullpage-hero__content__intro__eyebrow"
+				>
+					{{ slice.primary.eyebrowHeadline }}
+				</p>
+				<PrismicRichText
```
Comment (xrutayisire) [inline packages/adapter-nuxt/public/Hero/javascript.vue:33]: No need to import these Prismic components. How does it work?

File: packages/adapter-nuxt/public/Hero/javascript.vue [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+</template>
+
+<style>
+.es-bounded {
+	margin: 0px;
+	min-width: 0px;
+	position: relative;
+}
+
+.es-bounded-content {
```
Comment (xrutayisire) [inline packages/adapter-nuxt/public/Hero/javascript.vue]: Same, not used

File: packages/adapter-nuxt/public/Hero/javascript.vue (hunk 2 above)
Comment (hypervillain) [reply packages/adapter-nuxt/public/Hero/javascript.vue:8]: Screenshot 2023-09-27 at 14 59 42

I'm still very unclear about this, they're displayed correctly in my editor

**Pull request merged**